test/language/number/scientific_missing_fractional_part.wren

```diff
@@ -0,0 +1 @@
+var x = 1.e // expect error

```

Comment:
It looks like this test isn't failing at compile time. It's parsing this as `(1).e`, which makes sense. That allows things like `123.abs` to work so I think the behavior is right here.

You should just be able to change this to expect a runtime error to occur.